### PR TITLE
[JENKINS-55087] - Revert JAXB and Temporary update Center patches from the Java 11 image

### PR DESCRIPTION
**Pull request by oleg-nenashev**
[JENKINS-55087](https://issues.jenkins-ci.org/browse/JENKINS-55087)
Jenkins 2.163 introduced JAXB as a detached plugin. After this change there is no reason to bundle JAXB in the Docker image OR to define modules.

Also, the experimental update center patch can be reverted now - Pipeline: Support 3.0 resolves the issue

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
CC @jenkinsci/java11-support

**Comment by oleg-nenashev** [comment]
Works on my machine (c)

**Comment by oleg-nenashev** [comment]
It was. But it was configured in the script directly. And I deduplicated
the run scripts

File: Dockerfile-jdk11 [hunk 1]
```diff
@@ -67,8 +58,7 @@ RUN curl -fsSL ${JENKINS_URL} -o /usr/share/jenkins/jenkins.war \
   && echo "${JENKINS_SHA}  /usr/share/jenkins/jenkins.war" | sha256sum -c -
 
 ENV JENKINS_UC https://updates.jenkins.io
-# TODO: revert before the GA release of Java 11 support (JENKINS-55087)
-ENV JENKINS_UC_EXPERIMENTAL=https://updates.jenkins.io/temporary-experimental-java11
+ENV JENKINS_UC_EXPERIMENTAL=https://updates.jenkins.io/experimental
```
Comment (MRamonLeon) [inline Dockerfile-jdk11]: is this env prop needed anymore?

**Comment by oleg-nenashev** [comment]
yes. It is a part of the standard image, which allows installing plugins
from Experimental in plugins.txt

File: Dockerfile-jdk11 (hunk 1 above)
Comment (batmat) [reply Dockerfile-jdk11]: Yes, this is a revert to the previous value.

File: Dockerfile-jdk11 [hunk 2]
```diff
@@ -79,11 +69,12 @@ EXPOSE ${http_port}
 EXPOSE ${agent_port}
 
 ENV COPY_REFERENCE_FILE_LOG $JENKINS_HOME/copy_reference_file.log
+ENV JENKINS_OPTS --enable-future-java
```
Comment (batmat) [inline Dockerfile-jdk11]: NIT: maybe 

`ENV JENKINS_ENABLE_FUTURE_JAVA=true`

I tend to prefer env vars for this things nowadays, but don't feel that strongly in that very case.

**Comment by oleg-nenashev** [comment]
It should also resolve https://issues.jenkins-ci.org/browse/JENKINS-55391

File: Dockerfile-jdk11 (hunk 2 above)
Comment (oleg-nenashev) [reply Dockerfile-jdk11]:
```suggestion
ENV JENKINS_ENABLE_FUTURE_JAVA=true
```

Comment (oleg-nenashev) [reply Dockerfile-jdk11]: I agree that an Env Var is better here.
Once https://github.com/jenkinsci/extras-executable-war/pull/22 is integrated into weekly, we will be able to get rid of it anyway

**Comment by oleg-nenashev** [comment]
@batmat @carlossg I would like to proceed with merge. Is it fine with you?

**Comment by batmat** [comment]
@oleg-nenashev Given I had already approved, and you only added https://github.com/jenkinsci/docker/pull/798/commits/7e7503ded895b4735792443d6b975ab707e9660f in the meantime, you didn't need my approval anymore from my perspective.

But in case there's a doubt, yes, it is good to go :). :ship: :it: :wink:

**Comment by oleg-nenashev** [comment]
trying to retrigger CI, but it will unlikely help. Something is wrong with `docker` labels on the CI instance

**Comment by batmat** [comment]
Green, merging! \o/ thanks Oleg!

**Pull request merged**